### PR TITLE
Add RWKV benchmark

### PR DESCRIPTION
**Pull request by breuleux**
Benchmark based on this:

https://github.com/BlinkDL/RWKV-LM

This is an interesting LLM architecture that is based on RNNs rather than transformers. This is an opportunity to have an RNN in the benchmarks.

Based on #88

### CODE REVIEW

**Comment by bouthilx** [comment]
Does it fit on 1 GPU? Based on the repo's doc RWKV-4 has 7.4B parameters. OPT-6.7B in comparison does not fit on 80GB RAM, when including gradients, activations, optimizer state and data.

**Comment by breuleux** [comment]
@bouthilx It's configurable. I used one of the examples they gave that is configured to be smaller than that.

**Comment by breuleux** [comment]
It fails on ROCm while trying to compile a torch extension, unfortunately, so I think this will be CUDA-only for the foreseeable future.

**Comment by bouthilx** [comment]
C'est quoi HIP?

**Comment by Delaunay** [comment]
https://github.com/ROCm-Developer-Tools/HIP

**Comment by Delaunay** [comment]
C'est le cuda de ROCm

**Comment by bouthilx** [comment]
Je suis pas certain lequel serait le plus important à ajouter entre celui-ci et Whisper. Whisper permet de couvrir un domaine différent, mais c'est encore un transformer, alors que RWKV permet d'ajouter un RNN sans transformer, mais ça reste du language modeling. Je dirais que le plus facile à ajouter devrait être priorisé.

**Comment by bouthilx** [comment]
(Incluant support AMD)

**Comment by Delaunay** [comment]
Looking for ptxas ? I think that is redistributed with CUDA.
but if you install pytorch triton ptxas might get redistributed through it as well https://github.com/pytorch/builder/issues/1318

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -58,6 +58,10 @@ jobs:
           miniconda-version: "latest"
           activate-environment: test
 
+      - name: check
+        run: |
+          ls -l /usr/local/ /opt
```
Comment (Delaunay) [inline .github/workflows/tests.yml:63]: Remaining from debugging ?

**Review by Delaunay — APPROVED** [review]

**Comment by breuleux** [comment]
> Is there change we need to bring to the docker file ?

Yes, the change is already in the PR: https://github.com/mila-iqia/milabench/pull/90/files#diff-9d577c160bdb1afcf223f0f34f8194206efabfd0bda9a1629c52673b3997c5ebR44

**Pull request merged**